### ISSUE TITLE
看理想app或者网站节目无法收听

### ISSUE THREAD

**Issue by InnocenseYu** [Loyalsoldier/v2ray-rules-dat#293]
## 看理想app或者[网站免费节目](https://www.vistopia.com.cn/detail/11)播放没声音，无法播放，配置路由如下

`xray 配置文件`
```
{
    "log": {
        "loglevel": "warning"
    },

    "dns": {
        "hosts": {
            "domain:googleapis.cn": "googleapis.com",
            "dns.google": "8.8.8.8",
            "dns.pub": "119.29.29.29",
            "dns.alidns.com": "223.5.5.5",
            "geosite:category-ads-all": "127.0.0.1"
        },
        "servers": [
            {
                "address": "https://1.1.1.1/dns-query",
                "domains": [
                    "geosite:geolocation-!cn",
                    "geosite:google@cn"
                ],
                "expectIPs": [
                    "geoip:!cn"
                ],
                "skipFallback": false
            },
            {
                "address": "8.8.8.8",
                "port": 53,
                "skipFallback": false
            },
            {
                "address": "223.5.5.5",
                "port": 53,
                "domains": [
                    "geosite:cn",
                    "geosite:icloud",
                    "geosite:category-games@cn",
                    "ntp.org"
                ],
                "expectIPs": [
                    "geoip:cn"
                ],
                "skipFallback": true
            },
            {
                "address": "114.114.114.114",
                "port": 53,
                "domains": [
                    "geosite:cn",
                    "ntp.org"
                ],
                "skipFallback": true
            },
            {
                "address": "localhost",
                "skipFallback": true
            }
        ]
    },

    "inbounds": [
        {
            "tag": "all-in",
            "port": 23630,
            "protocol": "dokodemo-door",
            "settings": {
                "network": "tcp,udp",
                "followRedirect": true
            },
            "sniffing": {
                "enabled": true,
                "destOverride": [
                    "http",
                    "tls",
                    "quic"
                ]
            },
            "streamSettings": {
                "sockopt": {
                    "tproxy": "tproxy"
                }
            }
        }
    ],

    "routing": {
        "domainMatcher": "hybrid",
        "domainStrategy": "IPIfNonMatch",
        "rules": [
            {
                "type": "field",
                "protocol": [
                    "bittorrent"
                ],
                "outboundTag": "direct"
            },
            {
                "type": "field",
                "inboundTag": [
                    "all-in"
                ],
                "port": 123,
                "network": "udp",
                "outboundTag": "direct"
            },
            {
                "type": "field",
                "inboundTag": [
                    "all-in"
                ],
                "port": 53,
                "network": "udp",
                "outboundTag": "dns-out"
            },
            {
                "type": "field",
                "domain": "geosite:category-ads-all",
                "outboundTag": "block"
            },
            {
                "type": "field",
                "ip": [
                    "223.5.5.5",
                    "119.29.29.29",
                    "114.114.114.114"
                ],
                "outboundTag": "direct"
            },
            {
                "type": "field",
                "ip": [
                    "1.1.1.1",
                    "1.0.0.1",
                    "8.8.8.8",
                    "8.8.4.4"
                ],
                "outboundTag": "out"
            },
            {
                "type": "field",
                "domain": [
                    "geosite:geolocation-!cn"
                ],
                "outboundTag": "out"
            },
            {
                "type": "field",
                "domain": [
                    "geosite:tld-cn",
                    "geosite:category-games@cn"
                ],
                "outboundTag": "direct"
            },
            {
                "type": "field",
                "domain": [
                    "geosite:cn",
                    "geosite:private"
                ],
                "outboundTag": "direct"
            },
            {
                "type": "field",
                "ip": [
                    "geoip:cn", 
                    "geoip:private"
                ],
                "outboundTag": "direct"
            },
            {
                "type": "field",
                "network": "tcp,udp",
                "outboundTag": "out"
            }
        ]
    },

    "outbounds": [
        {
            "tag": "out",
            "protocol": "vless",
            "settings": {
                "vnext": [
                    {
                        "address": "xxxxxx",
                        "port": 443,
                        "users": [
                            {
                                "id": "xxxxxxxxxx",
                                "flow": "xtls-rprx-vision",
                                "encryption": "none",
                                "level": 0
                            }
                        ]
                    }
                ]
            },
            "streamSettings": {
                "sockopt": {
                    "mark": 255
                },
                "network": "tcp",
                "security": "tls",
                "tlsSettings": {
                    "serverName": "my.domain",
                    "allowInsecure": false,
                    "fingerprint": "chrome"
                }
            }
        },
        {
            "tag": "dns-out",
            "protocol": "dns",
            "streamSettings": {
                "sockopt": {
                    "mark": 255
                }
            }
        },
        {
            "tag": "direct",
            "protocol": "freedom",
            "settings": {
                "domainStrategy": "UseIPv4"
            },
            "streamSettings": {
                "sockopt": {
                    "mark": 255
                }
            }
        },
        {
            "tag": "block",
            "protocol": "blackhole",
            "settings": {
                "response": {
                    "type": "http"
                }
            }
        }
    ]
}
```

**Comment by Loyalsoldier** [comment]
看 Xray 日志文件

**Comment by InnocenseYu** [comment]
> 看 Xray 日志文件

- 会不会是dns设置的原因啊？使用以上配置文件，http协议打不开google，国内网站和**看理想**都正常；socks协议google 正常，**看理想**播放有问题。
- 对udp协议不懂，耽误您时间了，帮忙来回测试的，太感谢了。希望大佬帮忙看看配置文件，指导下，谢谢啦

## 操作播放页面时xray日志
```
Thu May 18 00:34:00 2023 daemon.info xray[3537]: 2023/05/17 16:34:00 [Warning] [1807152370] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:34:00 2023 daemon.info xray[3537]: 2023/05/17 16:34:00 192.168.1.46:1079 accepted tcp:13.107.5.80:443 [all-in -> us-out]
Thu May 18 00:34:03 2023 daemon.info xray[3537]: 2023/05/17 16:34:03 [Warning] [584110974] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:34:35 2023 daemon.info xray[3537]: 2023/05/17 16:34:35 192.168.1.46:1080 accepted tcp:162.159.137.105:443 [all-in -> tun-out]
Thu May 18 00:34:41 2023 daemon.info xray[3537]: 2023/05/17 16:34:41 192.168.1.44:63849 accepted udp:192.168.1.45:53 [all-in -> dns-out]
Thu May 18 00:34:41 2023 daemon.info xray[3537]: 2023/05/17 16:34:41 192.168.1.44:55688 accepted udp:192.168.1.45:53 [all-in -> dns-out]
Thu May 18 00:34:41 2023 daemon.info xray[3537]: 2023/05/17 16:34:41 192.168.1.46:1082 accepted tcp:13.107.5.80:443 [all-in -> us-out]
Thu May 18 00:34:45 2023 daemon.info xray[3537]: 2023/05/17 16:34:45 [Warning] [497605941] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:34:45 2023 daemon.info xray[3537]: 2023/05/17 16:34:45 192.168.1.46:1084 accepted tcp:13.107.5.80:443 [all-in -> us-out]
Thu May 18 00:34:48 2023 daemon.info xray[3537]: 2023/05/17 16:34:48 [Warning] [2192712351] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.44:63777 accepted udp:192.168.1.45:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.44:64754 accepted tcp:17.57.145.164:5223 [all-in -> direct]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:50027 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:60721 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:52281 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:47431 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:58251 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:54744 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:43185 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:40430 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:57 2023 daemon.info xray[3537]: 2023/05/17 16:34:57 192.168.1.45:43422 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:57 2023 daemon.info xray[3537]: 2023/05/17 16:34:57 192.168.1.45:35244 accepted udp:192.168.1.1:53 [all-in -> dns-out]
Thu May 18 00:34:58 2023 daemon.info xray[3537]: 2023/05/17 16:34:58 192.168.1.46:55718 accepted udp:192.168.1.45:53 [all-in -> dns-out]
Thu May 18 00:34:58 2023 daemon.info xray[3537]: 2023/05/17 16:34:58 [Error] app/dns: failed to retrieve response for client.wns.windows.com. > Post "https://1.1.1.1/dns-query": context canceled
Thu May 18 00:34:58 2023 daemon.info xray[3537]: 2023/05/17 16:34:58 192.168.1.46:1085 accepted tcp:20.197.71.89:443 [all-in -> direct]
Thu May 18 00:34:59 2023 daemon.info xray[3537]: 2023/05/17 16:34:59 192.168.1.46:1086 accepted tcp:20.24.121.134:443 [all-in -> direct]
Thu May 18 00:35:04 2023 daemon.info xray[3537]: 2023/05/17 16:35:04 192.168.1.46:1087 accepted tcp:163.181.35.230:443 [all-in -> tun-out]
Thu May 18 00:35:05 2023 daemon.info xray[3537]: 2023/05/17 16:35:05 192.168.1.46:1088 accepted tcp:163.181.35.230:443 [all-in -> tun-out]
Thu May 18 00:35:12 2023 daemon.info xray[3537]: 2023/05/17 16:35:12 192.168.1.46:1090 accepted tcp:13.107.5.80:443 [all-in -> us-out]
Thu May 18 00:35:15 2023 daemon.info xray[3537]: 2023/05/17 16:35:15 [Warning] [1957943063] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:35:15 2023 daemon.info xray[3537]: 2023/05/17 16:35:15 192.168.1.46:1092 accepted tcp:13.107.5.80:443 [all-in -> us-out]
Thu May 18 00:35:18 2023 daemon.info xray[3537]: 2023/05/17 16:35:18 [Warning] [4048797345] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:35:29 2023 daemon.info xray[3537]: 2023/05/17 16:35:29 192.168.1.46:1093 accepted tcp:13.107.5.80:443 [all-in -> us-out]
Thu May 18 00:35:32 2023 daemon.info xray[3537]: 2023/05/17 16:35:32 [Warning] [2133055707] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:35:32 2023 daemon.info xray[3537]: 2023/05/17 16:35:32 192.168.1.46:1094 accepted tcp:13.107.5.80:443 [all-in -> us-out]
Thu May 18 00:35:35 2023 daemon.info xray[3537]: 2023/05/17 16:35:35 [Warning] [2841538072] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
Thu May 18 00:36:06 2023 daemon.info xray[3537]: 2023/05/17 16:36:06 192.168.1.46:1095 accepted tcp:13.107.21.239:443 [all-in -> direct]
Thu May 18 00:36:09 2023 daemon.info xray[3537]: 2023/05/17 16:36:09 192.168.1.46:1096 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
Thu May 18 00:36:09 2023 daemon.info xray[3537]: 2023/05/17 16:36:09 192.168.1.46:1097 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1098 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1099 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1100 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1101 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
Thu May 18 00:36:11 2023 daemon.info xray[3537]: 2023/05/17 16:36:11 192.168.1.46:1102 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
Thu May 18 00:36:11 2023 daemon.info xray[3537]: 2023/05/17 16:36:11 192.168.1.46:1103 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
Thu May 18 00:36:14 2023 daemon.info xray[3537]: 2023/05/17 16:36:14 192.168.1.46:1105 accepted tcp:13.107.5.80:443 [all-in -> us-out]
```

**Comment by InnocenseYu** [comment]
> > 看 Xray 日志文件
> 
> * 会不会是dns设置的原因啊？使用以上配置文件，http协议打不开google，国内网站和**看理想**都正常；socks协议google 正常，**看理想**播放有问题。
> * 对udp协议不懂，耽误您时间了，帮忙来回测试的，太感谢了。希望大佬帮忙看看配置文件，指导下，谢谢啦
> 
> ## 操作播放页面时xray日志
> ```
> Thu May 18 00:34:00 2023 daemon.info xray[3537]: 2023/05/17 16:34:00 [Warning] [1807152370] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:34:00 2023 daemon.info xray[3537]: 2023/05/17 16:34:00 192.168.1.46:1079 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> Thu May 18 00:34:03 2023 daemon.info xray[3537]: 2023/05/17 16:34:03 [Warning] [584110974] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:34:35 2023 daemon.info xray[3537]: 2023/05/17 16:34:35 192.168.1.46:1080 accepted tcp:162.159.137.105:443 [all-in -> tun-out]
> Thu May 18 00:34:41 2023 daemon.info xray[3537]: 2023/05/17 16:34:41 192.168.1.44:63849 accepted udp:192.168.1.45:53 [all-in -> dns-out]
> Thu May 18 00:34:41 2023 daemon.info xray[3537]: 2023/05/17 16:34:41 192.168.1.44:55688 accepted udp:192.168.1.45:53 [all-in -> dns-out]
> Thu May 18 00:34:41 2023 daemon.info xray[3537]: 2023/05/17 16:34:41 192.168.1.46:1082 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> Thu May 18 00:34:45 2023 daemon.info xray[3537]: 2023/05/17 16:34:45 [Warning] [497605941] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:34:45 2023 daemon.info xray[3537]: 2023/05/17 16:34:45 192.168.1.46:1084 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> Thu May 18 00:34:48 2023 daemon.info xray[3537]: 2023/05/17 16:34:48 [Warning] [2192712351] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.44:63777 accepted udp:192.168.1.45:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.44:64754 accepted tcp:17.57.145.164:5223 [all-in -> direct]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:50027 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:60721 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:52281 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:47431 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:58251 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:54744 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:43185 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:56 2023 daemon.info xray[3537]: 2023/05/17 16:34:56 192.168.1.45:40430 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:57 2023 daemon.info xray[3537]: 2023/05/17 16:34:57 192.168.1.45:43422 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:57 2023 daemon.info xray[3537]: 2023/05/17 16:34:57 192.168.1.45:35244 accepted udp:192.168.1.1:53 [all-in -> dns-out]
> Thu May 18 00:34:58 2023 daemon.info xray[3537]: 2023/05/17 16:34:58 192.168.1.46:55718 accepted udp:192.168.1.45:53 [all-in -> dns-out]
> Thu May 18 00:34:58 2023 daemon.info xray[3537]: 2023/05/17 16:34:58 [Error] app/dns: failed to retrieve response for client.wns.windows.com. > Post "https://1.1.1.1/dns-query": context canceled
> Thu May 18 00:34:58 2023 daemon.info xray[3537]: 2023/05/17 16:34:58 192.168.1.46:1085 accepted tcp:20.197.71.89:443 [all-in -> direct]
> Thu May 18 00:34:59 2023 daemon.info xray[3537]: 2023/05/17 16:34:59 192.168.1.46:1086 accepted tcp:20.24.121.134:443 [all-in -> direct]
> Thu May 18 00:35:04 2023 daemon.info xray[3537]: 2023/05/17 16:35:04 192.168.1.46:1087 accepted tcp:163.181.35.230:443 [all-in -> tun-out]
> Thu May 18 00:35:05 2023 daemon.info xray[3537]: 2023/05/17 16:35:05 192.168.1.46:1088 accepted tcp:163.181.35.230:443 [all-in -> tun-out]
> Thu May 18 00:35:12 2023 daemon.info xray[3537]: 2023/05/17 16:35:12 192.168.1.46:1090 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> Thu May 18 00:35:15 2023 daemon.info xray[3537]: 2023/05/17 16:35:15 [Warning] [1957943063] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:35:15 2023 daemon.info xray[3537]: 2023/05/17 16:35:15 192.168.1.46:1092 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> Thu May 18 00:35:18 2023 daemon.info xray[3537]: 2023/05/17 16:35:18 [Warning] [4048797345] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:35:29 2023 daemon.info xray[3537]: 2023/05/17 16:35:29 192.168.1.46:1093 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> Thu May 18 00:35:32 2023 daemon.info xray[3537]: 2023/05/17 16:35:32 [Warning] [2133055707] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:35:32 2023 daemon.info xray[3537]: 2023/05/17 16:35:32 192.168.1.46:1094 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> Thu May 18 00:35:35 2023 daemon.info xray[3537]: 2023/05/17 16:35:35 [Warning] [2841538072] app/proxyman/outbound: failed to process outbound traffic > proxy/vless/outbound: failed to find an available destination > common/retry: [dial tcp 34.94.220.173:443: connect: connection refused] > common/retry: all retry attempts failed
> Thu May 18 00:36:06 2023 daemon.info xray[3537]: 2023/05/17 16:36:06 192.168.1.46:1095 accepted tcp:13.107.21.239:443 [all-in -> direct]
> Thu May 18 00:36:09 2023 daemon.info xray[3537]: 2023/05/17 16:36:09 192.168.1.46:1096 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
> Thu May 18 00:36:09 2023 daemon.info xray[3537]: 2023/05/17 16:36:09 192.168.1.46:1097 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
> Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1098 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
> Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1099 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
> Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1100 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
> Thu May 18 00:36:10 2023 daemon.info xray[3537]: 2023/05/17 16:36:10 192.168.1.46:1101 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
> Thu May 18 00:36:11 2023 daemon.info xray[3537]: 2023/05/17 16:36:11 192.168.1.46:1102 accepted tcp:91.108.23.100:443 [all-in -> tw-out]
> Thu May 18 00:36:11 2023 daemon.info xray[3537]: 2023/05/17 16:36:11 192.168.1.46:1103 accepted tcp:91.108.23.100:80 [all-in -> tw-out]
> Thu May 18 00:36:14 2023 daemon.info xray[3537]: 2023/05/17 16:36:14 192.168.1.46:1105 accepted tcp:13.107.5.80:443 [all-in -> us-out]
> ```

## 使用xray pc本地测试的，配置配置文件入站和路由有部分更改
```
"inbounds": [
      {
        "tag": "socks-in",
        "listen": "127.0.0.1",
        "port": 23630,
        "protocol": "socks",
        "settings": {
          "udp": true
        },
        "sniffing": {
          "enabled": true,
          "destOverride": [
            "http",
            "tls"
          ]
        }
      },
      {
        "tag": "http-in",
        "listen": "127.0.0.1",
        "port": 23631,
        "protocol": "http",
        "sniffing": {
          "enabled": true,
          "destOverride": [
            "http",
            "tls"
          ]
        }
      }
    ],

    "routing": {
      "domainStrategy": "IPIfNonMatch",
      "rules": [
        {
          "type": "field",
          "inboundTag": [
            "socks-in", 
            "http-in"
          ],
          "port": 53,
          "network": "udp",
          "outboundTag": "dns-out"
        },
...
```

**Comment by InnocenseYu** [comment]
在路由规则geosite的大分类 cn或者 tld-cn之前 domian:volcengineapi.com 使用直连即可
